src/Mod/Sketcher/Gui/Utils.cpp

```diff
@@ -791,12 +789,12 @@ std::string SketcherGui::angleToDisplayFormat(double value, int digits)
         QString escapeSecond = QString::fromLatin1("\"");          // substitute ascii double quote
         QString displayString = qUserString.replace(schemeMinute, escapeMinute);
         displayString = displayString.replace(schemeSecond, escapeSecond);
-        return Base::Tools::toStdString(displayString);
+        return displayString.toStdString();
     }
 
     // we always use use U+00B0 (°) as the unit of measure for angles in
     // single unit schema.  Will need a change to support rads or grads.
-    auto qUnitString = QString::fromUtf8("°");
+    std::string UnitString = "°";
```

Comment:
Do you know of the degree symbol is still displayed correctly?

Reply:
It is at least on Linux. `QString::fromUtf8` needs proper unicode representation in the source code, so I guess it should be correct on other platforms as well, but I have no chance to verify that myself.

Reply:
OK, we should keep an eye on this once the PR is merged.